waterloop-site/src/components/SustainableTech/Footer/STEFooter.tsx

```diff
@@ -22,17 +22,27 @@ const FooterBackground = styled.div`
   padding: 2.5rem 6rem 7.5rem;
   transition: all 0.3s;
   border: none !important;
+  @media screen and (max-width: 1024px) {
+    min-height: 15vh;
+  }
   @media screen and (max-width: 425px) {
     padding: 1rem;
-    min-height: 70vh;
+    min-height: 30vh;
     overflow: hidden;
   }
+  @media screen and (max-width: 375px) {
+    min-height: 35vh;
+  }
 `;
 
 const LogoAndContactWrapper = styled.div`
   display: flex;
   flex-direction: row;
   justify-content: start;
+  @media screen and (max-width: 425px) {
+    padding-top: 3vh;
+    padding-bottom: 2vh;
```

Comment:
How come `padding` was used here rather than `margin`? Although very similar, `margin` seems more appropriate to me. Also, can the same margin be used for top and bottom, i.e. replacing lines 43 and 44 with `margin: 2vh 0;`?